backend/mailer.js

```diff
@@ -18,10 +18,16 @@ exports.sendEmail = function (data) {
 
   var previewURL = 'http://' + hostname + '/preview/' + data.email + '/' + data.uniqueId + '_preview';
   var downloadURL = 'http://' + hostname + '/download/' + data.email + '/' + data.uniqueId;
-  var deployURL = 'http://' + hostname + '/github?email=' + data.email + '&uniqueId=' + data.uniqueId + '&gitURL=' + data.gitUrl;
+  var deployURL
```

Comment:
do not use `undefined` variables. Initialize it with an empty string. Also don't forget to add `;` at the end of a statement